maven/deps.bzl

```diff
@@ -0,0 +1,26 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def deps():
+    http_archive(
+        name = "rules_jvm_external",
+        strip_prefix = "rules_jvm_external-3.2",
+        sha256 = "82262ff4223c5fda6fb7ff8bd63db8131b51b413d26eb49e3131037e79e324af",
+        url = "https://github.com/bazelbuild/rules_jvm_external/archive/3.2.zip",
+    )
+    http_archive(
+        name = "io_bazel_rules_kotlin",
+        urls = ["https://github.com/bazelbuild/rules_kotlin/archive/legacy-1.3.0.zip"],
+        type = "zip",
+        strip_prefix = "rules_kotlin-legacy-1.3.0",
+        sha256 = "4fd769fb0db5d3c6240df8a9500515775101964eebdf85a3f9f0511130885fde",
+    )
+
+maven_artifacts = [
```

Comment:
can we remove `maven_artifacts` and only keep `maven_artifacts_with_version`?

Reply:
@lolski `maven_artifacts` are kept such that we can load them in our repos with the same setup we already have, like this:

```
load("@graknlabs_bazel_distribution//maven:deps.bzl", graknlabs_bazel_distribution_maven_artifacts = "maven_artifacts")
load("@graknlabs_dependencies//library/maven:rules.bzl", "maven")
maven(
    graknlabs_grabl_tracing_artifacts +
    graknlabs_graql_artifacts +
    graknlabs_dependencies_tool_maven_artifacts +
    graknlabs_client_java_artifacts +
    graknlabs_bazel_distribution_maven_artifacts,
    graknlabs_client_java_overrides
)

```

Reply:
Got it